webpack/loaders.js

```diff
@@ -37,27 +37,34 @@ exports.ts_JiT = {
 exports.istanbulInstrumenter = {
   enforce: 'post',
   test: /^(.(?!\.spec))*\.ts$/,
-  loader: 'istanbul-instrumenter-loader',
+  use: ['istanbul-instrumenter-loader'],
 };
 
 exports.html = {
   test: /\.html$/,
-  loader: 'raw-loader',
+  use: ['raw-loader'],
 };
 
 exports.localCss = {
   test: /\.css$/,
   include: path.resolve(process.cwd(), 'src', 'app'),
-  loader: 'to-string-loader!css-loader?-minimize!postcss-loader',
+  use: [
+    'to-string-loader',
+    { loader: 'css-loader', options: { minimize: true } },
```

Comment:
I believe that we had actually disabled `minimize` in `css-loader` because we are using `css-nano`